hw3/01-characters.js

```diff
@@ -1,4 +1,44 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
 
-fetch(url);
+const parentDiv = document.createElement('div');
+parentDiv.classList.add('parent-div-01');
+
+const section = document.getElementById('section-01');
+section.appendChild(parentDiv);
+
+function createCharacterCard(character) {
+  const cardDiv = document.createElement('div');
+  cardDiv.classList.add('card-div-01');
+
+  const image = document.createElement('img');
+  image.classList.add('dp-01');
+  image.setAttribute('src', character.imageUrl);
+  image.setAttribute('alt', `Display picture of ${character.fullName}`);
+
+  const name = document.createElement('h2');
+  name.classList.add('full-name-01');
+  name.innerText = character.fullName;
+
+  const title = document.createElement('h3');
+  title.classList.add('title-01');
+  title.innerText = character.title;
+
+  cardDiv.appendChild(image);
+  cardDiv.appendChild(name);
+  cardDiv.appendChild(title);
```

Comment:
Using the regular append() method all three child elements (image, name, and title) could be appended to cardDiv at the same time. This is not necessary, but it would save a few lines while still being readable.

The append() method works like appendChild(), but lets you pass in multiple items.

```suggestion
  cardDiv.append(image, name, title);
```